index.d.ts

```diff
@@ -516,7 +516,8 @@ declare global {
     readonly height: number
     constrainProportions: boolean
 
-    layoutAlign: "MIN" | "CENTER" | "MAX" | "STRETCH" // applicable only inside auto-layout frames
+    layoutAlign: "MIN" | "CENTER" | "MAX" | "STRETCH" | "INHERIT" // applicable only inside auto-layout frames
```

Comment:
Is there a good way to mark just `MIN`, `CENTER`, and `MAX` as deprecated here? or is this fine since they're marked in the docs?